xml/System.Windows.Media.Animation/Storyboard.xml

```diff
@@ -1748,16 +1748,16 @@
         <Parameter Name="element" Type="System.Windows.DependencyObject" />
       </Parameters>
       <Docs>
-        <param name="element">The dependency object from which to get the <see cref="F:System.Windows.Media.Animation.Storyboard.TargetProperty" />.</param>
-        <summary>Retrieves the <see cref="F:System.Windows.Media.Animation.Storyboard.TargetProperty" /> value of the specified <see cref="T:System.Windows.Media.Animation.Timeline" />.</summary>
+        <param name="element">The dependency object from which to get the <xref data-throw-if-not-resolved="true" uid="System.Windows.Media.Animation.Storyboard.TargetProperty_a"></xref>.</param>
+        <summary>Retrieves the <xref data-throw-if-not-resolved="true" uid="System.Windows.Media.Animation.Storyboard.TargetProperty_a"></xref> value of the specified <see cref="T:System.Windows.Media.Animation.Timeline" />.</summary>
         <returns>The property targeted by <paramref name="element" />.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method gets the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty> attached property on the specified object. For more information about how storyboard targeting works, see <xref:System.Windows.Media.Animation.Storyboard.TargetProperty> attached property. For more information about how attached properties work, see [Attached Properties Overview](/dotnet/framework/wpf/advanced/attached-properties-overview).  
+ This method gets the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%2A> attached property on the specified object. For more information about how storyboard targeting works, see <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%2A> attached property. For more information about how attached properties work, see [Attached Properties Overview](/dotnet/framework/wpf/advanced/attached-properties-overview).  
   
- Note that if the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty> was set in markup, this method returns an empty <xref:System.Windows.PropertyPath> because the property reference is resolved and stored internally when it is parsed.  
+ Note that if the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%2A> was set in markup, this method returns an empty <xref:System.Windows.PropertyPath> because the property reference is resolved and stored internally when it is parsed.  
```

Comment:
```suggestion
If the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a> was set in markup, this method returns an empty <xref:System.Windows.PropertyPath> because the property reference is resolved and stored internally when it is parsed.  
```

Reply:
I've revised the statement a bit.